corehq/apps/users/models.py

```diff
@@ -2050,6 +2055,12 @@ def __repr__(self):
             self=self
         ))
 
+    @skippable_quickcache(['self._id'], lambda _: settings.UNIT_TESTING)
```

Comment:
this on default i believe only caches for 5 minutes, we can make it longer right since this shouldn't change?